include/aws/cryptosdk/cipher.h

```diff
@@ -45,6 +45,10 @@ enum aws_cryptosdk_rsa_padding_mode {
     AWS_CRYPTOSDK_RSA_OAEP_SHA256_MGF1,
 };
 
+/* This is large enough to hold an encoded public key for all currently supported curves */
+#define MAX_PUBKEY_SIZE 64
```

Comment:
Do not move these to public headers. We don't want customers taking a dependency on these (they may change in the future as new signature algorithms are added) and they are not properly namespaced.

Reply:
Ok, I've reverted this change and moved the definition to `cipher_openssl.h` instead so it can still be accessed from the harnesses.